code/game/objects/effects/decals/Cleanable/tracks.dm

```diff
@@ -136,3 +136,8 @@ GLOBAL_LIST_EMPTY(fluidtrack_cache)
 	if(blood_state != C.blood_state) //We only replace footprints of the same type as us
 		return
 	..()
+
+/obj/effect/decal/cleanable/blood/footprints/can_bloodcrawl_in()
+	if(basecolor == COLOR_BLOOD_MACHINE)
```

Comment:
I don't really like these kind of indirect links.
It has the color X so it is Y.
Also maybe a bit out of the scope for this PR?

Reply:
Blood rites uses the bloodcrawl check to distinguish between oil and blood. hence the change. I wish there was a cleaner way to do this check but this would require refactoring pretty much from A to Z how blood tracks and splatters work which I really dont wanna get into.